target/linux/ipq40xx/base-files/etc/hotplug.d/firmware/11-ath10k-caldata

```diff
@@ -130,6 +130,7 @@ case "$FIRMWARE" in
 	;;
 "ath10k/pre-cal-ahb-a000000.wifi.bin")
 	case "$board" in
+	8dev,habanero-dvk |\
```

Comment:
Hi, since you are using 11-ath10k-caldata  here, I wanted to kindly ask you whether you could test my patchset for caldata extraction unification in PR #2464 on this device to cover ipq40xx.

You would just need the first four patches from my PR.

I'm just asking for a favor, so feel free to decline this request and just mark it as resolved then.

Reply:
Sorry, I would gladly test but I am spread so thin with time currently.